app/controllers/spaces_controller.rb

```diff
@@ -81,6 +81,33 @@ def destroy
     head :no_content
   end
 
+  # PATCH /spaces/1/enable_shareable_link
+  def enable_shareable_link
+    if @space.enable_shareable_link!
+      render json: SpaceRepresenter.new(@space).to_json(user_options: {current_user_can_edit: true}), status: :ok
```

Comment:
I'm a bit uncomfortable setting ``current_user_can_edit`` to true as opposed to finding this from the model methods.  Realize there's some redundancy, but still.